### DISCUSSION TITLE
Блокировка закрыть окно в режиме iframe

### DISCUSSION THREAD

**Discussion by vldekov** [totumonline/totum-ru-issues-and-discussions#334] [Как это сделать?] [answered]
Как можно запретить пользователю закрыть окно в режиме iframe?
Например, требуется ввести обязательный контроль заполнения всех полей в окне или выход только по кнопке с действием.
Вариант 1: скрыть х в окне.
Вариант 2: выход по кнопке.

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Нельзя запретить закрыть. Можно запретить выполнение действия, но запретить отмену — нет.

**Reply by vldekov** [reply]
возможно скрыть Х ( крестик в правом верхнем углу окна ) в окне iframe?

**Reply by ttm-alexeypolunin** [reply]
Только внизу — bottombuttons.

Дело в том, что вы так не заблокируете закрытие iframe — браузерное обновить продолжит работать ни в чем не бывало. Не стройте архитектуру таким образом.

Эта задача рассматривалась и записана как — «глобальное блокирование», не уверен, что мы будем когда-то это реализовывать, так-как этим легко привести систему в запредельный режим работы и у нас здесь чат будет забит вопросами «как разблокировать срочно без кредитки и смс».
